presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java

```diff
@@ -61,8 +61,16 @@ public class NativeExecutionSystemConfig
     private static final String NUM_IO_THREADS = "num-io-threads";
     private static final String PRESTO_VERSION = "presto.version";
     private static final String SHUTDOWN_ONSET_SEC = "shutdown-onset-sec";
+    // Memory related configurations.
     private static final String SYSTEM_MEMORY_GB = "system-memory-gb";
     private static final String QUERY_MEMORY_GB = "query.max-memory-per-node";
+    private static final String USE_MMAP_ALLOCATOR = "use-mmap-allocator";
+    private static final String ENABLE_MEMORY_ARBITRATION = "enable-memory-arbitration";
+    private static final String MEMORY_POOL_INIT_CAPACITY = "init-memory-pool-capacity";
+    private static final String MEMORY_POOL_TRANSFER_CAPACITY = "min-memory-pool-transfer-capacity";
```

Comment:
Any particular reason variable name doesn't match property name. The property name has min- prefix, but variable name doesn't.

Reply:
I am refactoring the presto cpp side. Updated.